web/src/types.ts

```diff
@@ -46,20 +44,4 @@ export type UpdateMessageEvent = {
     topic?: string;
 };
 
-// TODO/typescript: Move this to user_profile
```

Comment:
nit: Looks like this change does not belong in this commit -- is it misplaced?

Reply:
I've intentionally removed that because I've changed the function and that type wasn't used anywhere after that change. I've written a new type in the further commits that is more accurate.

Reply:
Makes sense. Thanks .